packages/orchestrator/internal/template/build/provision.sh

```diff
@@ -1,6 +1,15 @@
 #!/bin/bash
 set -euo pipefail
 
+# Use a temporary log file
+LOGFILE=$(mktemp)
+# Trap to clean up on exit
+trap 'rm -f "$LOGFILE"' EXIT
+# Start logging in the background, prefixing lines
+( tail -f "$LOGFILE" | sed -u 's/^/{{ .LogPrefixExternal }}/' ) &
+# Redirect all output to the logfile
+exec >"$LOGFILE" 2>&1
```

Comment:
The file tail leaves the tail as orphaned process and also I was getting:
```
tail: /var/folders/q3/nsq5lg4s1r93cwpy_d0139mh0000gn/T/tmp.57DgydKUG5: No such file or directory
```

```suggestion
LOGPIPE=$(mktemp -u)
echo "$LOGPIPE"
mkfifo "$LOGPIPE"
trap 'rm -f "$LOGPIPE"' EXIT

( sed -u 's/^/{{ .LogPrefixExternal }}/' < "$LOGPIPE" ) &
exec >"$LOGPIPE" 2>&1
```

Reply:
Where did you get this `tail: /var/folders/q3/nsq5lg4s1r93cwpy_d0139mh0000gn/T/tmp.57DgydKUG5: No such file or directory`?<br><br>Also, where do you see the orphaned process?